packages/ng/button/button.component.ts

```diff
@@ -37,13 +39,39 @@ export class ButtonComponent implements OnChanges {
 	 */
 	luButton: '' | 'outlined' | 'text' | 'text-invert' = '';
 
+	#iconComponentRef?: ElementRef;
+
+	@ContentChild(IconComponent, { read: ElementRef })
+	set iconComponentRef(ref: ElementRef) {
+		this.#iconComponentRef = ref;
+		this.updateClasses();
+	}
+
+	private get iconOnly(): boolean {
+		const listOfNode = Array.from((this.#elementRef?.nativeElement as HTMLButtonElement)?.childNodes || []);
```

Comment:
```suggestion
		const listOfNode = Array.from((this.#elementRef?.nativeElement)?.childNodes || []);
```
goes with the previous comment